specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,103 @@
+# Service Subcategory
+
+The Service Subcategory is a further classification of the Service Category for a [*service*](#glossary:service) based on its core function.  The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.  
```

Comment:
Is it really true that we are aiding analysis across providers if all providers are adhering to the same set of standard values. Isn't the use case more that it is aiding a practitioner in analyzing their cloud spend across specific workload types?

Reply:
I would argue: yes, absolutely! I think it's both/and instead of either/or. The goal is to aid a practitioner in analyzing their cloud spend and usage for specific workload types across providers.  Do you agree with that statement?

Reply:
"analyzing their cloud spend and usage for specific workload types across providers" - if we can update that portion to say when used in conjunction with the Provider column, then I would agree.

Reply:
Alright.  How about this:

```suggestion
The Service Subcategory is a secondary classification of the Service Category for a [*service*](#glossary:service) based on its core function. The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing spend and usage for specific workload types across providers and tracking the migration of workloads across fundamentally different architectures.  
```

Reply:
Looks good.